.github/stale.yml

```diff
@@ -0,0 +1,19 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 365
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 90
+# Limit to only `issues` or `pulls`
+only: issues
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - "Type: Feature"
+  - "Type: Understood"
+# Label to use when marking an issue as stale
+staleLabel: "Status: Stale"
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as "stale" because it has not had
+  any activity for a while. It will be closed in 90 days if no further activity occurs.
+  Thank you for your contributions!
+# Comment to post when closing a stale issue. Set to `false` to disable
+closeComment: true
```

Comment:
If we don't define `closeComment`, then we should not enable it.

I think `markComment` (comment on 365 days) is enough, don't want to spam project activity subscribers one more time.

Reply:
Ohh shot I was confusing things... >.<

Reply:
But Yes, I think if you see:
"This will be closed in 90 days"

and 90 days later:
"Issue is closed by bot" 

That its clear enough...